PORT SCANNER.PY

```diff
@@ -116,5 +116,5 @@ t2 = datetime.now()
 total =  t2 - t1
 
 # Printing the information to screen
-print 'Scanning Completed in: ', total
+print('Scanning Completed in: ', total)
```

Comment:
__print__ statements __that contain a comma__ need special treatment on Python 2.  
* __python2 -c "print('a', 'b')"__  # --> ('a', 'b')
* __python2 -c "from __future__ import print_function ; print('a', 'b')"__  # --> a b

Reply:
Thanks for your help